core/src/validator.rs

```diff
@@ -1861,7 +1861,7 @@ fn load_blockstore(
     }
 
     if !config.no_poh_speed_test {
-        check_poh_speed(&genesis_config, None)?;
+        check_poh_speed(&genesis_config, None).map_err(|err| format!("{err}"))?;
```

Comment:
This is kind of ugly, but I'm considering BP'ing the change so I decided to keep the diff minimal. In a subsequent PR, I will make the surrounding function also return `Result<_, ValidatorError>` such that the `map_err` goes away.